frontend/packages/ceph-storage-plugin/src/components/bucket-class/state.ts

```diff
@@ -10,7 +10,9 @@ export const initialState = {
   writeNamespaceStore: [],
   hubNamespaceStore: null,
   cacheBackingStore: null,
-  timeToLive: 3600000,
+  timeToLiveInMs: 3600000,
+  timeToLive: 1,
```

Comment:
So, first we convert given time to ms and then convert back ms to given time. Is it?

Reply:
The CR uses the time in milliseconds, so yes it gets converted to ms as per current code and various validations are done in ms.

You can however try changing to ms after the review page but it will require significant changes in validation code across the pages. 

It will be easier for you to just implement a simple func to convert from ms to given time unit.. Will save time for you in testing.